apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -232,7 +246,7 @@ const PaymentsTableContent: React.FC<IPaymentsTableProps> = ({ billingContext, c
                                 <Row justify='end' align='middle'>
                                     {
                                         appliedFiltersCount > 0 && (
-                                            <Col>
+                                            <Col onClick={() => filtersAreReset = true }>
```

Comment:
It's better to use the click handler of the reset button. You may try to wrap `handleResetSearch` into your own function.

Reply:
tried it, but because `handleResetSearch` is awaited, `useEffect` triggers first with no filters and still thinking that `filtersAreReset = false`. so it applies default filters, which shouldn’t be the case

Reply:
Ok, I've understood. Appropriate as a quick decision ✅.